### DISCUSSION TITLE
Can ccache support distcc localslots_cpp?

### DISCUSSION THREAD

**Discussion by zcfh** [ccache/ccache#1200] [Questions & answers] [answered]
From the distcc documentation, localslots_cpp is set to limit the number of concurrent processes doing preprocessing, but it doesn't seem to work when called via ccache.
Does ccache need to be set up by other means?

**Comment by afbjorklund** [comment, marked as answer]
If memory serves me, we added `CCACHE_PREFIX_CPP` for this use case.

3fbd20042187f29cf19282f3afb46c56db4155a3
